internal/build/job/maker.go

```diff
@@ -148,7 +148,7 @@ func (m *maker) specTemplate(
 
 func (m *maker) containerArgs(
 	buildConfig *kmmv1beta1.Build,
-	targetKernel string,
+	mld *api.ModuleLoaderData,
 	containerImage string,
 	registryTLS *kmmv1beta1.TLSOptions,
```

Comment:
registryTLS can also be obtained from mld.  It is passed from specTemplate function, so it can be removed from specTemnplate's arguments and obtained directly from mld

Reply:
correct, good catch!